sdk/core/azure-core/inc/azure/core/credentials/credentials.hpp

```diff
@@ -23,7 +23,7 @@ namespace Azure { namespace Core { namespace Credentials {
   /**
    * @brief An access token is used to authenticate requests.
    */
-  struct AccessToken final
+  struct AccessToken final // LCOV_EXCL_LINE
```

Comment:
Seems odd to exclude this line. Is it excluding an entire struct? Why just this type? This seems buggy.

Reply:
It is gcovr's false positives.